src/main/java/hudson/tasks/Mailer.java

```diff
@@ -371,16 +370,14 @@ private static Session createSession(String smtpHost, String smtpPort, boolean u
             	 * and thats done in mail sender, and it would be a bit of a hack to get it all to
             	 * coordinate, and we can make it work through setting mail.smtp properties.
             	 */
-            	if (props.getProperty(SMTP_SOCKETFACTORY_PORT_PROPERTY) == null) {
+            	if (props.getProperty(SMTP_SOCKETFACTORY_PORT_PROPERTY)==null) {
```

Comment:
Still, you removed the spaces around the `==`

```suggestion
            	if (props.getProperty(SMTP_SOCKETFACTORY_PORT_PROPERTY) == null) {
```